package.json

```diff
@@ -87,6 +85,10 @@
     "tiny-lr": "^1.1.1",
     "vinyl-source-stream": "^2.0.0"
   },
+  "dependencies": {
+    "nebenan-react-datepicker": "^1.0.0",
+    "nebenan-helpers": "^4.1.1"
```

Comment:
-> peerDependencies

do we even need it?

Reply:
nebenan-helpers is used in datepicker (dom, keymanager, eventproxy). 

Will add them to peerDependencies - but I don't fully get why. It's not like they are some kind of plugins. Wouldn't matter to have two different versions in dependency tree. 
Smaller bundle size?